README.md

```diff
@@ -232,9 +232,20 @@ Results and models are available in the [model zoo](docs/en/model_zoo.md).
 **Note:** All the about **300+ models, methods of 40+ papers** in 2D detection supported by [MMDetection](https://github.com/open-mmlab/mmdetection/blob/master/docs/en/model_zoo.md) can be trained or used in this codebase.
 
 ## Installation
-
-Please refer to [getting_started.md](docs/en/getting_started.md) for installation.
-
+MMDetection3D depends on [PyTorch](https://pytorch.org/) and [MMCV](https://github.com/open-mmlab/mmcv).
+Below are quick steps for installation.
+Please refer to [getting_started.md](docs/en/getting_started.md) for more details.
+```shell
```

Comment:
1. format issue, this does not follow the markdown format.
2. Suggest putting this section into getting_started.md, see https://github.com/open-mmlab/mmdetection/pull/7575